readme-mo.md

```diff
@@ -0,0 +1,24 @@
+## HTTP and the like
+![http1 vs http2](https://i.imgur.com/qPh7wFb.gif)
+
+## General
+
+This is a project that is trying to use HTTP/HTTP2 on the back-end.
+
+The project has been initialised using Yarn.
+
+## Getting started
+
+To get this project up and running:
+
+`git clone`
```

Comment:
What you usually see in reame's is that people use a code block to highlight the installment (this way you can see it at a glance). So something like this.
```
git clone
```